<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IW233ImgSpiderClass</class>
 <widget class="QWidget" name="IW233ImgSpiderClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>680</width>
    <height>630</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>IW233ImgSpider</string>
  </property>
  <widget class="QPushButton" name="pb_dir1">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>25</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>1</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_dir2">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>25</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>2</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_dir3">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>40</y>
     <width>25</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>3</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_dir4">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>70</y>
     <width>25</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>4</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+4</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="edit_dir1">
   <property name="geometry">
    <rect>
     <x>35</x>
     <y>40</y>
     <width>240</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="edit_dir2">
   <property name="geometry">
    <rect>
     <x>35</x>
     <y>70</y>
     <width>240</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="edit_dir3">
   <property name="geometry">
    <rect>
     <x>315</x>
     <y>40</y>
     <width>240</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="edit_dir4">
   <property name="geometry">
    <rect>
     <x>315</x>
     <y>70</y>
     <width>240</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_del">
   <property name="geometry">
    <rect>
     <x>569</x>
     <y>40</y>
     <width>101</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>DEL[Q]</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_tmp">
   <property name="geometry">
    <rect>
     <x>569</x>
     <y>70</y>
     <width>101</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>TMP[ ` ]</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+`</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="edit_url">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>265</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>https://iw233.cn/api.php?sort=random</string>
   </property>
   <property name="placeholderText">
    <string>ENTER URL HERE</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="edit_tmp">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>10</y>
     <width>265</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>ENTER TEMPDIR HERE</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_run">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>10</y>
     <width>101</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>RUN</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_img">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>600</width>
     <height>500</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>NO IMAGE</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_status">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>95</y>
     <width>661</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>STOPPED</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="pg_quality">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>570</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="maximum">
    <number>10000</number>
   </property>
   <property name="value">
    <number>0</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="invertedAppearance">
    <bool>false</bool>
   </property>
   <property name="textDirection">
    <enum>QProgressBar::BottomToTop</enum>
   </property>
   <property name="format">
    <string>%vk</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="pg_buffer">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>510</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="minimum">
    <number>0</number>
   </property>
   <property name="maximum">
    <number>10</number>
   </property>
   <property name="value">
    <number>1</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="invertedAppearance">
    <bool>false</bool>
   </property>
   <property name="textDirection">
    <enum>QProgressBar::BottomToTop</enum>
   </property>
   <property name="format">
    <string>%vp</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="pg_speed">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>450</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="maximum">
    <number>150</number>
   </property>
   <property name="value">
    <number>0</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="invertedAppearance">
    <bool>false</bool>
   </property>
   <property name="textDirection">
    <enum>QProgressBar::BottomToTop</enum>
   </property>
   <property name="format">
    <string>%v/m</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="IW233ImgSpider.qrc"/>
 </resources>
 <connections/>
</ui>
